test/scripts/pwrite.sh

```diff
@@ -1,9 +1,14 @@
-
 #!/bin/bash
 
+thread=$1
+count=$2
+size=$3 
+mntPath=$4 
+outputPath=$5 
+sed_line=$6
 
 start_time=`date +%s`
-time seq 1 $2 | parallel --will-cite -j $1 -I{} dd if=/dev/zero of=$4/$3_{}.tst bs=1M count=$3
+time seq 1 $count | parallel --will-cite -j $thread -I{} dd if=/dev/zero of=$mntPath/$size_{}.tst bs=1M count=$size
```

Comment:
Here replace /dev/zero with /dev/random. Write now this test case is generating a file which has only 0, which will be very fast in terms of generation and disk writing and does not qualify to be a real-life test case. /dev/random will generate random data.